content/api/commands/attachfile.md

```diff
@@ -0,0 +1,217 @@
+
+Pass in an options object to change the default behavior of `.attachFile()`.
+
+| Option              | Default                                                               | Description                                                                                                                                                         |
+| ------------------- | --------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `action`            | `'input'`                                                             | Switches attachFile between modes. Valid values are `input` and `drag-n-drop`. See [Actions](#Actions) below for more details.                                      |
+| `animationDistanceThreshold` | [`animationDistanceThreshold`](/guides/references/configuration#Actionability) | The distance in pixels an element must exceed over time to be [considered animating](/guides/core-concepts/interacting-with-elements#Animations). |
+| `force`             | `false`                                                               | Forces the action, disables [waiting for actionability](#Assertions)                                                                                                |
+| `log`               | `true`                                                                | Displays the command in the [Command log](/guides/core-concepts/test-runner#Command-Log)                                                                            |
+| `timeout`           | [`defaultCommandTimeout`](/guides/references/configuration#Timeouts)  | Time to wait for `.attachFile()` to resolve before [timing out](#Timeouts)                                                                                                |
+| `waitForAnimations` | [`waitForAnimations`](/guides/references/configuration#Actionability) | Whether to wait for elements to [finish animating](/guides/core-concepts/interacting-with-elements#Animations) before executing the command.                        |
+
+### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
+
+<List><li>`.attachFile()` yields the same subject it was given from the previous command.</li></List>
+
+### Action
+
+Depending on the action set in the `options` argument, `.attachFile()` can simulate two different user behaviors:
+
+- `input` (default)
+By default, `attachFile()` runs in 'input' mode, mimicking a user selecting one or more files on an HTML5 input element. In this mode, the subject [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management) must be a single `input` element with `type="file"`, or a `label` element connected to an input (either with its `for` attribute or by containing the input).
```

Comment:
This icon is not standard to use inline with sentences, we just link the word. 

```suggestion
By default, `attachFile()` runs in 'input' mode, mimicking a user selecting one or more files on an HTML5 input element. In this mode, the [subject](/guides/core-concepts/introduction-to-cypress#Subject-Management) must be a single `input` element with `type="file"`, or a `label` element connected to an input (either with its `for` attribute or by containing the input).
```

Reply:
> A few other things I noticed while reading through the updated preview.
> 
> Some of the code blocks are showing scrollbars at the max size. Can you update the code blocks so you don't have to scroll to read? See preview: [deploy-preview-4241--cypress-docs.netlify.app/api/commands/attachfile#Arguments](https://deploy-preview-4241--cypress-docs.netlify.app/api/commands/attachfile#Arguments)

Went through and fixed all those.

Reply:
Ah, whoop, last one wasn't supposed to be a child comment. Also fixed this.